### PR TITLE
Improve AntiBlackout

### PR DESCRIPTION
**Pull request by Tommy-XL**
Progress:
- [x] Rework the black screen protection warning message
- [x] Add notice message with information about exile player for vanilla players
- [x] Remove the "Black Screen Prevention (Recommended)" setting (or give the opportunity to completely disable any protection and sending warning messages)

### CODE REVIEW

File: Modules/ExtendedPlayerControl.cs [hunk 1]
```diff
@@ -426,8 +417,16 @@ public static void ResetPlayerCam(this PlayerControl pc, float delay = 0f)
...
         {
+            // For some reason, the players are protected after the meeting
+            // Need to remeve protection
+            if (!pc.IsProtected())
+            {
+                pc.RpcSpecificMurderPlayer(pc, pc);
+            }
+
             pc.RpcSpecificMurderPlayer(pc, pc);
         }, 0.2f + delay, "Murder To Reset Cam");
```
Comment (Tommy-XL) [inline Modules/ExtendedPlayerControl.cs:431]: @NikoCat233 The player has protect after the meeting.
But at the same time `RpcGuardAndKill();` not involved in this in any way, do you know the reason for this or is it happening on the AU side?

**Comment by NikoCat233** [comment]
If the player sometimes is protected and you want to force murder it
You just write flags.succeeded, without decisionbyhost

Decision by host is checked before succeeded , together with failed protected in vanilla code

**Comment by NikoCat233** [comment]
I'm not sure why the player is protected after meeting, I dont have my pc now.

**Comment by Tommy-XL** [comment]
Hmm, ok

**Comment by Tommy-XL** [comment]
Tasks:
[] - Rework the black screen protection warning
[] - Remove the "Black Screen Prevention (Recommended)" setting (or give the opportunity to completely disable any protection and sending warning messages)

**Comment by Tommy-XL** [comment]
Everything seems to be working fine

**Comment by Tommy-XL** [comment]
Tested it in the game and everything works fine

**Pull request merged**